cmd/state/internal/cmdtree/internal/middleware/befaft/befaft.go

```diff
@@ -0,0 +1,37 @@
+package befaft
```

Comment:
I'll surely make it longer, but this is due to the admonishment to keep pkg names short like "bufio" instead of "bufferinputoutput". :trollface: 

Also, general names like "events" leads to packages with expansive and ever-expanding dependencies. This is why "runners" contains specific packages like "run". In this case, the corollary to "runners" is "middleware".